Cargo.toml

```diff
@@ -198,3 +199,12 @@ vec_init_then_push = 'allow'
 while_let_on_iterator = 'allow'
 too_many_arguments = 'allow'
 clone_on_copy = 'allow'
+
+[patch.crates-io]
```

Comment:
Probably wasn't intended to get committed?

Reply:
Need this locally until I can land the gel-rust changes :)